lua/nvim-treesitter/parsers.lua

```diff
@@ -2167,6 +2167,14 @@ list.yuck = {
   maintainers = { "@Philipp-M", "@amaanq" },
 }
 
+list.zathurarc = {
+  install_info = {
+    url = "https://github.com/Freed-Wu/tree-sitter-zathurarc",
+    files = { "src/parser.c" },
+    branch = "main",
```

Comment:
No need for `branch` if it's the default

Reply:
It looks like default branch is master

Reply:
```
nvim-treesitter[zathurarc]: Failed to execute the following command:
{
  cmd = "mv",
  opts = {
    args = { "-f", "tree-sitter-zathurarc-tmp/tree-sitter-zathurarc-master", "tree-sitter-zathurarc" },
    cwd = "/home/wzy/.local/share/nvim",
    stdio = {
      [2] = <userdata 1>,
      [3] = <userdata 2>
    }
  }
}
mv: cannot stat 'tree-sitter-zathurarc-tmp/tree-sitter-zathurarc-master': No such file or directory
```

Reply:
But there is no `master` branch on github though ?

Reply:
The default branch for nvim-treesitter is indeed master (we may need to fix that at some point) but it will use the commit defined in the lockfile so this is still unnecessary.